misc/ExampleExtension.py

```diff
@@ -0,0 +1,76 @@
+import bpy
+import sys
+from io_scene_gltf2.io.com.gltf2_io_extensions import Extension
+from io_scene_gltf2.io.com import gltf2_io
+
+bl_info = {
+    "name": "KDAB_example_extension",
+    "category": "Generic",
+    "blender": (2,80,0),
+    'isDraft': False,
+    'developer': "KDAB",
+    'url': 'https://www.kdab.com',
+}
+
+class ExampleExtensionProperties(bpy.types.PropertyGroup):
+    enabled: bpy.props.BoolProperty(
+        name='ExampleExtension_enabled',
+        description='This is an example of a BoolProperty used by a UserExtension.',
+        default=True
+        )
+    float_property: bpy.props.FloatProperty(
+        name='ExampleExtension_enabled',
+        description='This is an example of a BoolProperty used by a UserExtension.',
+        default=1.0
+        )
+
+def register():
+    bpy.utils.register_class(GLTF_PT_export_example_extension)
+    bpy.utils.register_class(ExampleExtensionProperties)
+    bpy.types.Scene.ExampleExtensionProperties = bpy.props.PointerProperty(type=ExampleExtensionProperties)
+
+def unregister():
+    bpy.utils.unregister_class(GLTF_PT_export_example_extension)
+    bpy.utils.unregister_class(ExampleExtensionProperties)
+    del bpy.types.Scene.ExampleExtensionProperties
+
+class GLTF_PT_export_example_extension(bpy.types.Panel):
+
+    bl_space_type = 'FILE_BROWSER'
+    bl_region_type = 'TOOL_PROPS'
+    bl_label = "Example Extension"
+    bl_parent_id = "GLTF_PT_export_user_extensions"
```

Comment:
For the same reason that the glTF addon might not have loaded yet, you can't put the panel inside anything it will create. If you don't want it inside just `FILE_PT_operator`, gltf-blender-io and the user-extension will need to "talk" to each other somehow. Or else go back to loading them itself I guess.

Reply:
During the invoke method in gltf2 exporter, can I ask for the extensions to return the panels and add there th bl_parent_id?

Reply:
I think you can set it in the user extension but you can't register it until after GLTF_PT_export_user_extensions has been registered.

I'm not quite sure how unregistering would need to work though. I can't tell if gltf-blender-io would need to unregister any panels that had been parented to GLTF_PT_export_user_extensions in its unregister function or not.

Reply:
Should be ready now. When the exporter looks for the extension addons, it also registers the panels. This is done with a try block, because it may try to register two times a panel if you export two or more times. The unregister function also has a try block, because if you unregister the extension without exporting, then the panel is not registered and it fails.

Reply:
I'm not sure it's right, but if it works I guess it's fine.